src/features/game/events/landExpansion/replenishStamina.ts

```diff
@@ -74,6 +78,7 @@ export function replenishStamina({ state, createdAt }: Options): GameState {
   const stamina = calculateBumpkinStamina({
     nextReplenishedAt: createdAt,
     bumpkin,
+    boostMultiplier: tent ? 1.1 : 1,
```

Comment:
I didn't really like this ternary. Open to ideas

Reply:
How about extracting that into a local function?

Reply:
Agreed @Polysemouse 

Similar to how we handle other boosts, it would be useful to extract into a separate function. This will also make it easier to test when we add new boosts.

E.g.

```
function getStaminaBoosts(gameState: GameState) {
   let multiplier = 1;

   if (gameState.builds.Tent) {
      mulitplier += 0.1
   }

   return multiplier
}


...
const stamina = calculateBumpkinStamina({
    nextReplenishedAt: createdAt,
    bumpkin,
    boostMultiplier: getStaminaBoost(stateCopy),
...
```